build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.0'
+        classpath 'com.android.tools.build:gradle:3.1.1'
```

Comment:
gradle version is now 3.1.2

Reply:
I had to downgrade since compilation failed with lates. I'll give it another shot.